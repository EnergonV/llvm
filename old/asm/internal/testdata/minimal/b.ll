define i32 @main() {
	%foo = add i32 42, 42
	ret i32 %foo
}
