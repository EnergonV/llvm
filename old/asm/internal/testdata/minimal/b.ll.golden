define i32 @main() {
; <label>:0
	%foo = add i32 42, 42
	ret i32 %foo
}
